addons/stock/models/product.py

```diff
@@ -502,8 +502,6 @@ def action_open_quants(self):
                 default_product_id=self.id,
                 single_product=True
             )
-        else:
-            self = self.with_context(product_tmpl_id=self.product_tmpl_id.id)
```

Comment:
Why did you remove this part ?

Reply:
`self.product_tmpl_id.id` doesn't work with differents products (from the kit) and I also saw it's not used in the action